torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1277,19 +1284,17 @@ int listPop(Stack& stack) {
 
 template <typename T>
 int listClear(Stack& stack) {
-  c10::List<T> list;
-  pop(stack, list);
+  c10::List<T> list = pop(stack).to<c10::List<T>>();
 
   list.clear();
   return 0;
 }
 
 template <typename T>
 int listInsert(Stack& stack) {
-  c10::List<T> list;
-  int64_t idx;
-  T elem;
-  pop(stack, list, idx, elem);
+  T elem = pop(stack).to<T>();
+  int64_t idx = pop(stack).to<int64_t>();
+  c10::List<T> list = pop(stack).to<c10::List<T>>();
```

Comment:
any particular reason previous templated pop doesn't work here? Because the List<T> does seem to be typed

Reply:
GenericList doesn't have a default constructor anymore. You need to either supply the types or the `deprecatedUntypedList` tag.